playwright.config.ts

```diff
@@ -1,13 +1,30 @@
 import { defineConfig, devices } from '@playwright/test';
+import dotenv from 'dotenv';
+import path from 'path';
 
 /**
  * See https://playwright.dev/docs/test-configuration.
  */
 
+dotenv.config({ path: path.resolve(__dirname, '.env.local') });
```

Comment:
`.env.local` is only found on local machines, so perhaps should prioritise `.env` and then `.env.local`. I'm guessing this would not work in review pipeline as expected 🤔 

`path: ['.env', '.env.local']`